frontstage/templates/partials/useful_links.html

```diff
@@ -0,0 +1,43 @@
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+<div class="ons-grid__col ons-col-4@m ons-push-1@m">
+      {%
+                call onsRelatedContent({
+                    "ariaLabel": 'Related content',
+                    "rows": [
+                        {
+                            "id": "useful-links",
+                            "title": "Useful links",
+                            "itemsList": [
+                                {
+                                    "id": "add_survey_button",
+                                    "url": url_for("surveys_bp.add_survey"),
+                                    "text": "Add a new survey",
+                                },
+                                {
+                                    "url": url_for("account_bp.share_survey_overview"),
+                                    "text": "Share access to surveys",
+                                },
+                                {
+                                    "url": url_for("account_bp.transfer_survey_overview"),
+                                    "text": "Transfer surveys"
+                                },
+                                {
+                                    "url": url_for("contact_us_bp.send_message"),
```

Comment:
The docs have this as the old get help with this survey link, however have matched the current implementation of that link which takes you to the send message page.